better_profanity/utils.py

```diff
@@ -18,8 +18,14 @@ def read_wordlist(filename: str):
                 yield row
 
 
-def get_replacement_for_swear_word(censor_char):
-    return censor_char * 4
+def get_replacement_for_swear_word(censor_char, n=4):
```

Comment:
It's just my personal preference: could you replace `n` with a more detailed variable ? It's quite unclear what `n` is when we call `get_replacement_for_swear_word("-", n=2)`.

Reply:
Sure.